### DISCUSSION TITLE
Please consider changing the version logic

### DISCUSSION THREAD

**Discussion by RaenonX** [tailwindlabs/headlessui#3318] [Ideas]
The fact that `enterFrom` and `enterTo`, etc. just disappeared / stopped working on `<Transition/>` because of the v2.1.0 doesn't make too much sense to me.

According to https://semver.org/#summary, _MINOR version when you add functionality in a backward compatible manner._ The behavior I mentioned above shouldn't happen given that it's a minor version bump.

**Comment by RobinMalfait** [comment]
Hey!

We only added new functionality and didn't remove functionality so it's still following semver correctly. 

Here is a minimal reproduction where you can see that the props didn't disappear and that everything still works as expected: https://codesandbox.io/p/devbox/musing-fermi-43ymvj

If that's not the case for you, can you open an issue with a minimal reproduction repo attached?

**Reply by RaenonX** [reply]
Hi, thanks for the response! I'll try to reproduce whenever available. The break happened with `<TransitionChild/>` .
